### PR TITLE
holo-hosting release 

### PR DESCRIPTION
**Pull request by zo-el**
This is the work that was dev tested on #575

### CODE REVIEW

**Review by zippy — APPROVED** [review]

File: profiles/logical/hpos/default.nix [hunk 1]
```diff
@@ -170,6 +175,16 @@ in
...
+
+        "/hc/master/" = {
+          proxyPass = "http://127.0.0.1:42211/";
+          proxyWebsockets = true;
+        };
+
+        "/hc/admin/" = {
+          proxyPass = "http://127.0.0.1:42233/";
+          proxyWebsockets = true;
+        };
```
Comment (peeech) [inline profiles/logical/hpos/default.nix:187]: Quick question - which service is making use of those endpoints?

Comment (zo-el) [reply profiles/logical/hpos/default.nix:187]: `/hc/master/`: This port is used by tryorama when it tries to connect to a holoport to get consistency information.
`/hc/admin`: I am not sure.

@mjbrisebois is there any use of those endpoints in the hosted pipeline?

Comment (mjbrisebois) [reply profiles/logical/hpos/default.nix:187]: No, these are not supposed to stay in hpos.  They are needed to inspect conductor during debugging from outside the OS, but they are security holes that cannot stay there.

**Review by peeech — APPROVED** [review]
Now it looks great!

**Pull request merged**